catalogue/webapp/services/prismic/types/event.ts

```diff
@@ -0,0 +1,27 @@
+import { Contributor, Image } from '@weco/common/model/catalogue';
+import { RichTextField } from '@prismicio/types';
+import { ContentType } from './index';
+
+export type Event = {
+  image: {
+    url: string;
+  };
+  summary: Standfirst[];
+  id: string;
+  contributors: Contributor[];
+  firstPublicationDate: Date;
+  title: Title;
+  type: ContentType[];
+};
+
+export type Title = {
+  text: {
+    text: RichTextField;
+  };
+};
+
+export type Standfirst = {
```

Comment:
Title and Standfirst are repeated throughout, maybe they could be in the index file? (But also maybe we wait until we're happy with all of them before simplified and I hold my horses)